src/VCAuthn/Views/Authorize/Authorize.cshtml

```diff
@@ -0,0 +1,101 @@
+        <path id="Fill-1" d="M467.621,40.551h7.667V7.672h10.469V0H457.152V7.672h10.469V40.551" transform="translate(-361.607)" fill="#fff" fill-rule="evenodd"/>
+        <path id="Fill-3" d="M672.583,40.551h7.667V7.672H690.72V0H662.114V7.672h10.469V40.551" transform="translate(-523.732)" fill="#fff" fill-rule="evenodd"/>
+        <path id="Fill-4" d="M247.9,40.551l10.611-22.346,10.164,22.346H277.1L258.66,0,239.418,40.551H247.9" transform="translate(-189.38)" fill="#fff" fill-rule="evenodd"/>
+        <path id="Fill-5" d="M901.568,40.551,892.054,24.1A13.468,13.468,0,0,0,896.9,13.84C896.9,6.211,890.335,0,882.262,0h-6.7V7.672h6.7q.361,0,.712.032c3.512.316,6.262,2.947,6.262,6.135s-2.75,5.819-6.262,6.136q-.352.032-.712.032h-2.546v7.667h2.546a15.476,15.476,0,0,0,2.85-.268l7.6,13.144h8.86" transform="translate(-692.568)" fill="#fff" fill-rule="evenodd"/>
+        <path id="Fill-6" d="M24.358,13.038l-5.637,5.615L0,0V40.551H7.671V18.474l11.05,11.007L29.772,18.474V40.551h7.672V0L24.358,13.038" fill="#fff" fill-rule="evenodd"/>
+      </g>
+      <g id="Group_28" data-name="Group 28" transform="translate(-1 -8)">
+        <g id="Group_26" data-name="Group 26" transform="translate(112)">
+          <g id="iphone_6:7_black" data-name="iphone 6:7 black" transform="translate(2333.094 909) rotate(90)">
+            <rect id="Rectangle_1" data-name="Rectangle 1" width="325" height="703.094" rx="58" transform="translate(-808 1582)" fill="#fff" stroke="#707070" stroke-width="1"/>
+            <rect id="IMG_4428" width="301.671" height="536.572" rx="4" stroke-width="1" fill="#fff" stroke="#707070" transform="translate(-795.933 1663.25)"/>
+          </g>
+        </g>
+      </g>
+      <g id="Group_29" data-name="Group 29" transform="translate(-84)">
+        <text id="matter.global" transform="translate(773 465)" fill="#fff" font-size="32" font-family="HelveticaNeue, Helvetica Neue"><tspan x="-92.768" y="0">matter.global</tspan></text>
+        <g id="globe" transform="translate(648 440)">
+          <circle id="Ellipse_7" data-name="Ellipse 7" cx="12" cy="12" r="12" transform="translate(2 2)" stroke-width="2" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" fill="none"/>
+          <line id="Line_15" data-name="Line 15" x2="24" transform="translate(2 14)" fill="none" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/>
+          <path id="Path_15" data-name="Path 15" d="M12,2a20.486,20.486,0,0,1,4,12,20.486,20.486,0,0,1-4,12A20.486,20.486,0,0,1,8,14,20.486,20.486,0,0,1,12,2Z" transform="translate(2)" fill="none" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/>
+        </g>
+      </g>
+      <g id="Group_30" data-name="Group 30">
+        <text id="_mattrglobal" data-name="@("@")mattrglobal" transform="translate(347 465)" fill="#fff" font-size="32" font-family="HelveticaNeue, Helvetica Neue"><tspan x="-93.552" y="0">@("@")mattrglobal</tspan></text>
```

Comment:
OK to leave Mattr in here?

Reply:
You're right, we might need to re-skin the whole view.

Reply:
removed mattr references